app/controllers/councils_controller.rb

```diff
@@ -0,0 +1,59 @@
+class CouncilsController < ApplicationController
+  def index
+    @councils = Council.where(user_id: current_user.id).order('created_at DESC')
+  end
+
+  def new
+    @council = Council.new
+  end
+
+  def create
+
+    @council = Council.where(council_params).first_or_create
+    if @council.save
+      if request.xhr?
+        render :json => @council
+      else
+      flash[:notice] = 'Added to list'
+      redirect_to councils_path
+      end
+    else
+      flash[:error] = 'Field cannot be left blank.'
+      render :new
+    end
+  end
+
+  def show
+    find_council(params[:id])
```

Comment:
If you are calling find_council in many places, you could put it on a before_action in the controller. So you could have 
before_action :find_council, only [:show, :edit, :update, :destroy]
and remove the find_council calls from those methods. Then edit and show would be empty so you can just remove them.